src/libraries/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SystemEvents.cs

```diff
@@ -517,7 +517,7 @@ private static void EnsureRegisteredSessionNotification()
             }
         }
 
-        private static UserPreferenceCategory GetUserPreferenceCategory(int msg, IntPtr wParam, IntPtr lParam)
+        private static UserPreferenceCategory GetUserPreferenceCategory(int msg, nint wParam, IntPtr lParam)
```

Comment:
`wParam` is unsigned in native, so technically `nuint` is a better match.

Reply:
This is eventually passes to a public API as IntPtr argument https://github.com/dotnet/runtime/blob/7993e515586bb644d85f6f3b408900dc065f96fd/src/libraries/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/TimerElapsedEventArgs.cs#L18